pkg/ovs/ovsctl/ovsctl.go

```diff
@@ -398,9 +398,15 @@ func (c *ovsCtlClient) DumpPortsDesc() ([][]string, error) {
 
 func (c *ovsCtlClient) SetPortNoFlood(ofport int) error {
 	// This command does not have standard output, and only has standard err when running with error.
-	_, err := c.ovsOfctlRunner.RunOfctlCmd("mod-port", strconv.FormatUint(uint64(ofport), 10), "no-flood")
+	// Note that, THIS CONFIGURATION MUST WORK WITH OpenFlow10.
```

Comment:
> just a question, looking at the changes in PR4517, could the bug also be caused by not capturing the standard err? 
Impressive observation, I had already taken this into account, but I found that there is no any stdout or stderr when running cmd `ovs-ofctl mod-port br-int 6 no-flood -O Openflow15`, and no-flood is not set.

> because otherwise before the change we could use c.runOfctlCmd(false, "mod-port", ofport) [to avoid Openflow15](https://github.com/NamanAg30/antrea/blob/7ec0860b5245a8c1e344477a534fcfe684eabdab/pkg/ovs/ovsctl/ofctl.go#L166).

Thanks for reminding, we can use `runOfctlCmd` here.